### PR TITLE
iOS ffmpeg build

### PR DESCRIPTION
**Pull request by jstys92**
Adds a script for building ios arm64

### CODE REVIEW

**Comment by DillonSkeehan** [comment]
What would our standard flow be for updating ffmpeg for ios ? Just build arm64?

**Comment by jstys92** [comment]
Yeah that's all we build for on ios, it's the same in fafnir

**Comment by jstys92** [comment]
Updated this with an rpath change for shared libs - should probably actually merge it this time around

File: build_ios/build-ffmpeg-iOS-framework.sh [hunk 1]
```diff
@@ -0,0 +1,191 @@
+#!/bin/sh
```
Comment (DillonSkeehan) [inline build_ios/build-ffmpeg-iOS-framework.sh:1]: did you write these build script files or did you grab them from somewhere?

Comment (jstys92) [reply build_ios/build-ffmpeg-iOS-framework.sh:1]: I grabbed them, the source is in the build_ios/readme: https://github.com/kewlbear/FFmpeg-iOS-build-script

**Pull request merged**